packages/kit/src/utils/routing.js

```diff
@@ -79,6 +88,13 @@ export function parse_route_id(id) {
 	return { pattern, names, types };
 }
 
+/**
+ * @param {string} segment
+ */
```

Comment:
Shouldn't we collapse this?
```suggestion
/** @param {string} segment */
```

Reply:
For types on variables I think that's prefereable, but for functions I'd like to keep it that way.